.cirrus.yml

```
@@ -2,15 +2,25 @@ env:
   ELIXIR_ASSERT_TIMEOUT: 2000
   LANG: C.UTF-8
 
```

I think there is no need to clone the whole repo:

```suggestion
  CIRRUS_CLONE_DEPTH: 1

```

More info at: https://cirrus-ci.org/guide/writing-tasks/#behavioral-environment-variables

---

So true. 
done!